Javastructure.js

```diff
@@ -0,0 +1,98 @@
+const cardData = [
+  {
+    question: 'Warum sind Eichhörnchen eigentlich gar nicht so süß?',
+    answer:
+      'Eichhörnchen sind in Wirklichkeit keine süßen kleinen Nussknacker, sondern Raubtiere und Nesträuber, die sich hauptsächlich von Eiern und kleinen Vögeln ernähren.',
+    isBookmarked: true,
```

Comment:
I think having ` isBookmarked: false` per default is a bit more logical than having it `true`. Would be like every post on Instagram or Facebook is per default liked and you have to unlike them... Does that make sense?